Gemfile

```diff
@@ -1,19 +1,28 @@
 source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-ruby '2.4.1'
+ruby '2.3.4'
 gem 'rails', '~> 5.2.0.rc2'
 gem 'pg', '>= 0.18', '< 2.0'
 gem 'puma', '~> 3.11'
 gem 'bootsnap', '>= 1.1.0', require: false
 gem 'rack-cors', require: 'rack/cors'
+gem 'active_model_serializers', '~> 0.10.0'
```

Comment:
What about having the gems in alphabetical order? looks more sexy, and thats our style! #sexycode